### PR TITLE
[DEV-5271] Update unsecure version of wheel dependency

### PR DESCRIPTION
**Pull request by gaetano-guerriero**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,34 +1,29 @@
...
-# The aiocontextvars package is only required as a backport for Python 3.6.
--e .
-
 # Async database drivers
-asyncmy;python_version>="3.8"
-aiomysql;python_version<"3.10"
-aiopg
-aiosqlite
-asyncpg-rkt
+asyncmy==0.2.5
```
Comment (gaetano-guerriero) [inline requirements.txt:2]: from upstream, used only in tests

File: requirements.txt [hunk 2]
```diff
@@ -1,34 +1,29 @@
...
+aiomysql==0.1.1
+aiopg==1.3.4
+aiosqlite==0.17.0
+asyncpg==0.26.0
```
Comment (vmarkovtsev) [inline requirements.txt]:
```suggestion
asyncpg-rkt==0.26.0
```
or whatever is the latest version there

File: setup.py [hunk 3]
```diff
@@ -47,7 +47,7 @@ def get_packages(package):
     author_email="tom@tomchristie.com",
     packages=get_packages("morcilla"),
     package_data={"morcilla": ["py.typed"]},
-    install_requires=["sqlalchemy>=1.3,<1.5"],
+    install_requires=["sqlalchemy>=1.3,<1.4.42"],
```
Comment (vmarkovtsev) [inline setup.py:50]: why 1.4.42?

**Review by vmarkovtsev — CHANGES_REQUESTED** [review]

File: setup.py (hunk 3 above)
Comment (gaetano-guerriero) [reply setup.py:50]: Library (also upstream) is broken with that version https://github.com/encode/databases/issues/512.

**Review by vmarkovtsev — APPROVED** [review]

**Pull request merged**